src/main/java/athleticli/ui/NutrientVerifier.java

```diff
@@ -0,0 +1,27 @@
+package athleticli.ui;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Verify the nutrient from a list of approved nutrients to be log in diet and diet goals
+ */
+public class NutrientVerifier {
+    public static final Set<String> VERIFIED_NUTRIENTS = new HashSet<>(
+            Arrays.asList("fats", "carb", "protein", "calories"));
```

Comment:
Suggestion: create unmodifiable set with `Set.of` introduced in Java 9.
```suggestion
    public static final Set<String> VERIFIED_NUTRIENTS = Set.of("fats", "carb", "protein", "calories");
```